runelite-client/src/test/java/net/runelite/client/plugins/mining/MiningPluginTest.java

```diff
@@ -149,4 +165,30 @@ private void testMessage(String gameMessage)
 		miningPlugin.onChatMessage(chatMessage);
 		assertNotNull(miningPlugin.getSession());
 	}
-}
\ No newline at end of file
+
+	@Test
+	public void testCamTorumWaterNotify()
+	{
+		when(client.getLocalPlayer()).thenReturn(mock(Player.class));
+		when(client.getLocalPlayer().getWorldLocation()).thenReturn(new WorldPoint(1501, 9538, 1));
+
+		GameStateChanged gameStateChanged = new GameStateChanged();
+		gameStateChanged.setGameState(GameState.LOGGED_IN);
+		miningPlugin.onGameStateChanged(gameStateChanged);
+		when(miningConfig.camTorumNotifyWaterSpawn()).thenReturn(true);
```

Comment:
Is this intending to test enabling the config after logging in? If so, some kind of `verify()` call should be made before changing the config value.